### PR TITLE
[Edit] JavaScript: Loops

### PR DESCRIPTION
**Pull request by ValeQRamos**
### Description

Adding the output to the first example in javascript/loops 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/loops/loops.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ while (i < 5) {
   console.log(i);
   i++;
 }
+
+// Output: 0 1 2 3 4 5
 ```
```
Comment (Dusch4593) [inline content/javascript/concepts/loops/loops.md:40]: Let's actually add a separate \```shell snippet with the output between lines # 39 and # 41.

```
0
1
2
3
4
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Hi @ValeQRamos thank you for your interest in contributing to Codecademy Docs! 😄

I just finished taking a look at your changes and had some feedback/suggestions to offer. I also wanted to suggest the following:

- On line # 3, can we shorten the `Description` to (ideally) 150 characters or less?
- In the first sentence on line # 16, can we **bold** the word "loop"?

File: content/javascript/concepts/loops/loops.md (hunk 1 above)
Comment (ValeQRamos) [reply content/javascript/concepts/loops/loops.md:40]: Sure, I got you

**Review by Dusch4593 — APPROVED** [review]
Thank you, @ValeQRamos, and Happy New Year! This looks good for the next review 😄

**Review by KTom101 — CHANGES_REQUESTED** [review]
@ValeQRamos I've done a second review and added a few comments.

**Comment by KTom101** [comment]
@ValeQRamos Just checking in to see if you need help implementing anything.

**Review by KTom101 — APPROVED** [review]
@ValeQRamos I went ahead and made all the necessary changes (in commits [7014108](https://github.com/Codecademy/docs/pull/1345/commits/701410831a9d929ad30db41bbc66a9aa5c727ca6) & [741f7d5](https://github.com/Codecademy/docs/pull/1345/commits/741f7d5b1a978f9d4006536144948b15230338df)) due to inactivity.

**Comment by KTom101** [comment]
@ValeQRamos Thank you for contributing! You are now listed as a contributor and can see the live changes here: https://www.codecademy.com/resources/docs/javascript/loops

Screen Shot 2023-01-30 at 12 51 41 PM

**Comment by KTom101** [comment]
cc @yangc95

**Pull request merged**